domain/src/main/java/org/oppia/domain/util/StateRetriever.kt

```diff
@@ -222,15 +224,21 @@ class StateRetriever @Inject constructor(
   }
 
   // Creates VoiceoverMappings from JSON and adds onto State
-  private fun createVoiceOverMappingsFromJson(recordedVoiceovers: JSONObject, stateBuilder: State.Builder) {
+  private fun createVoiceOverMappingsFromJson(
```

Comment:
Are all these changes in formatting coming after running ktlint? If not, then please run ktlint on this file, I want to make sure we are following the rules that ktlint has specified

Reply:
done